TypeScript browser bundle has webpack warning

<!--

BEFORE SUBMITTING AN ISSUE:

1.  Search for your issue on GitHub: https://github.com/prettier/prettier/issues
    A large number of opened issues are duplicates of existing issues.
    If someone has already opened an issue for what you are experiencing,
    you do not need to open a new issue — please add a 👍 reaction to the
    existing issue instead.

2.  If your issue is with a prettier editor extension or add-on, please open the
    issue in the repo for that extension or add-on, instead of this repo.

-->

**Environments:**
- Prettier Version: 1.19.0 and 1.19.1
- Running Prettier via: Browser API<!-- CLI, Node.js API, Browser API, etc. -->
- Runtime: webpack 4.41.2<!-- Node.js v6, Chrome v67, etc. -->
- Operating System: macOS<!-- Windows, Linux, macOS, etc. -->

**Steps to reproduce:**
<!-- shell script, js code, or a link to the minimal reproducible repository -->
```js
import "prettier/parser-typescript";
```

**Expected behavior:**
Bundle continues to compile without warnings.

**Actual behavior:**
```
WARNING in ./node_modules/prettier/parser-typescript.js
Module not found: Error: Can't resolve '@microsoft/typescript-etw' in '/Users/nick/Repos/prettier-chrome-extension/node_modules/prettier'
 @ ./node_modules/prettier/parser-typescript.js
 @ ./src/content/parsers.js
 @ ./src/content/github.js
 @ ./src/content/extension.js
 @ ./src/content/index.js
```
Oddly enough, the parser continues to work properly. However the error is confusing and may cause potential issues as `typescript-etw` only supports running on Windows natively, and this version of the parser only supports running in web browsers.

--- a/scripts/build/config.js
+++ b/scripts/build/config.js
@@ -41,12 +41,6 @@ const parsers = [
     input: "src/language-js/parser-typescript.js",
     target: "universal",
     babelPlugins: [babelReplaceArrayIncludesWithIndexof],
-    commonjs: {
-      ignore: [
-        // Optional package for TypeScript that logs ETW events (a Windows-only technology).
-        "@microsoft/typescript-etw"
-      ]
-    },
     replace: {
       'require("@microsoft/typescript-etw")': "undefined"
     }

--- a/scripts/build/config.js
+++ b/scripts/build/config.js
@@ -46,6 +46,9 @@ const parsers = [
         // Optional package for TypeScript that logs ETW events (a Windows-only technology).
         "@microsoft/typescript-etw"
       ]
+    },
+    replace: {
+      'require("@microsoft/typescript-etw")': "undefined"
     }
   },
   {

